calamari_ocr/ocr/predictor.py

```diff
@@ -262,9 +262,10 @@ def batched_data_params():
             for batch in progress_bar_wrapper(batched_data_params()):
                 sample_ids, batch_images, batch_params = zip(*batch)
                 samples = [dataset.samples()[i] for i in sample_ids]
+                current_mode = dataset.mode 
                 with ExitStack() as stack:
                     raw_dataset = [
-                        stack.enter_context(RawInputDataset(DataSetMode.PREDICT,
+                        stack.enter_context(RawInputDataset(current_mode, #DataSetMode.PREDICT,
```

Comment:
remove the comment, `current_mode` is fine

Reply:
done